common/src/main/java/org/broadleafcommerce/common/resource/service/ResourceMinificationServiceImpl.java

```diff
@@ -94,6 +95,19 @@ public Resource minify(Resource originalResource) {
             LOG.warn("Attempted to modify resource without a filename, returning non-minified resource");
             return originalResource;
         }
+
+        Integer filenameLength = originalResource.getFilename().length();
+
+        if (filenameLength >= JS_MIN.length()) {
+
+            String lastSevenCharacters = originalResource.getFilename().substring(filenameLength-JS_MIN.length());
+            if (lastSevenCharacters.equals(JS_MIN)) {
+                LOG.info("Minification already have done for this resource: " + originalResource.getFilename());
+                return originalResource;
+            }
+
```

Comment:
We don't need the check here because we're just delegating to the other `minify` method anyways. All this method does is make sure that the file has a name before delegating